src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -33,11 +42,66 @@ def cli_asset():
     default=False,
 )
 @_handle_error
-def asset_snapshot(**args):
+def asset_snapshot(root_dir, manifest_out, recursive, **args):
     """
     Creates manifest of files specified root directory.
     """
-    click.echo("snapshot taken")
+    root_dir_basename = os.path.basename(root_dir) + "_"
```

Comment:
Right, I chose not to do so because `_write_local_input_manifest` is used by other parts of the code, and they already have a standard for passing the parameter name of manifests. I'm adding the '_' to make a new name for manifests, and that only happens in snapshot. E.g. "stangch_assets" + "_" + "manifests" for snapshot, while job history only uses it while named "manifests"
It makes sense to move it closer to where it actually gets used though, so I moved it down ?